shim/src/sanitize/accessors.js

```diff
@@ -0,0 +1,62 @@
+// Adapted from SES/Caja - Copyright (C) 2011 Google Inc.
```

Comment:
noticed that most files are using 4 spaces, and this one is using 2, maybe we should just add a editorconfig file to the project.